gradle.properties

```diff
@@ -16,6 +16,13 @@ org.gradle.jvmargs=-Xmx1536m
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
 
+# AndroidX package structure to make it clearer which packages are bundled with the
+# Android operating system, and which are packaged with your app's APK
+# https://developer.android.com/topic/libraries/support-library/androidx-rn
+android.useAndroidX=true
+# Automatically convert third-party libraries to use AndroidX
+android.enableJetifier=true
```

Comment:
I'm not sure I want the behaviour of `android.enableJetifier`. It's often hard enough to reproduce issues as it is.

Reply:
Hi @wdullaer - Oh that's interesting.. Do you maybe have a link to more information on the behavior you mentioned? I have not seen any issues with it, and it's part of the default template of Android Studio. I'm also curious to find out if there are problems. Thanks!